app/views/employers/employer_profiles/_index_table.html.erb

```diff
@@ -36,7 +36,7 @@
       <% else %>
       <%- if er.active_broker_agency_account.present? %>
         <% broker_agency_profile = er.active_broker_agency_account.broker_agency_profile %>
-        <td><%= link_to broker_agency_profile.legal_name, broker_agencies_profile_path(broker_agency_profile) %></td>
+        <td><%#= link_to broker_agency_profile.legal_name, broker_agencies_profile_path(broker_agency_profile) %></td>
```

Comment:
Let's remove the commented-out code for now to keep our code clean. We can reintroduce the link if needed in the future